chains/solana/contracts/programs/ccip-router/src/lib.rs

```diff
@@ -206,6 +206,34 @@ pub mod ccip_router {
         v1::admin::update_svm_chain_selector(ctx, new_chain_selector)
     }
 
+    /// Bumps the CCIP version for a destination chain.
+    /// This effectively just resets the sequence number of the destination chain state.
+    /// If there had been a previous rollback, on re-upgrade the sequence number will resume from where it was
+    /// prior to the rollback.
+    ///
+    /// # Arguments
+    /// * `ctx` - The context containing the accounts required for the bump.
+    /// * `dest_chain_selector` - The destination chain selector to bump version for.
+    pub fn bump_ccip_version_for_dest_chain(
+        ctx: Context<UpdateDestChainSelectorConfig>,
+        dest_chain_selector: u64,
+    ) -> Result<()> {
+        v1::admin::bump_ccip_version_for_dest_chain(ctx, dest_chain_selector)
+    }
+
+    /// Rolls back the CCIP version for a destination chain.
+    /// This effectively just restores the old version's sequence number of the destination chain state.
+    ///
+    /// # Arguments
+    /// * `ctx` - The context containing the accounts required for the rollback.
+    /// * `dest_chain_selector` - The destination chain selector to rollback the version for.
+    pub fn rollback_ccip_version_for_dest_chain(
```

Comment:
Seeing as this is the part that will be seen from cargo docs, it may be good to include here an explanation about the consecutive rollback limitation. Especially because the error message you get isn't necessarily too explici.